rest/permissions.py

```diff
@@ -114,3 +114,19 @@ def has_permission(self, request, view):
                 RepoPermission.manage_taxonomy[0]
                 in get_perms(request.user, repo)
             )
+
+
+class ManageRepoMembersPermission(BasePermission):
+    """Checks manage_repo_users permission"""
+
+    def has_permission(self, request, view):
+        if request.method in SAFE_METHODS:
+            return True
+        try:
+            repo = get_repo(view.kwargs['repo_slug'], request.user.id)
+        except NotFound:
+            raise Http404()  # pragma: no cover
```

Comment:
the three `pragma: no cover`s are in section of the code that will never reached because the same checks run in other sections of the code, but I still did not want to remove the code because (like in this case) if we use this permission without others, the check matters.
To properly test this I should write, for example, a test api view with only this check and assert there is 404, but honestly, I'm not going to do it this week. If you want we can open a new Issue for this.